libs/blockscout-service-launcher/src/launcher/settings.rs

```diff
@@ -50,6 +50,7 @@ pub struct HttpServerSettings {
     pub addr: SocketAddr,
     pub max_body_size: usize,
     pub cors: CorsSettings,
+    pub base_path: Option<String>,
```

Comment:
`actix_web::web::scope` mentions not to add trailing slashes to the scope; let's also mention it somewhere here for easier configuration

Reply:
added validation